doc/release-notes.md

```diff
@@ -62,6 +62,19 @@ Due to a backward-incompatible change in the wallet database, wallets created
 with version 0.16.0 will be rejected by previous versions. Also, version 0.16.0
 will only create hierarchical deterministic (HD) wallets.
 
+Custom wallet directories
+---------------------
+The ability to specify a directory other than the default data directory in which to store
+wallets has been added. An existing directory can be specified using the `-walletdir=<dir>`
+argument. Wallets loaded via `-wallet` arguments must be in this wallet directory. Care should be taken
+when choosing a wallet directory location, as if  it becomes unavailable during operation,
+funds may be lost.
+
+On new installations (if the data directory doesn't exist), this parameter will default to a
```

Comment:
I think this is a very important change from the viewpoint of users ("where's my wallet.dat?! all wiki articles say it's in the data directory!"), and probably warrants its own section.

Reply:
Agreed, although I would hope most users would work out what the new "wallets/" directory was for ;)